.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,24 @@
+name: golangci-lint
+on:
+  push:
+    branches: ["master","dev"]
+      
+  pull_request:
+permissions:
+  contents: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
```

Comment:
we are using 1.19, could you bump the number

Reply:
sure